packages/qwik/src/cli/new/run-new-command.ts

```diff
@@ -26,16 +26,22 @@ export async function runNewCommand(app: AppCommand) {
     const args = app.args.filter((a) => !a.startsWith('--'));
 
     const mainInput = args.slice(1).join(' ');
-    let typeArg: 'route' | 'component' | undefined = undefined;
+    let typeArg: 'route' | 'component' | 'markdown' | undefined = undefined;
     let nameArg: string | undefined;
     let outDir: string | undefined;
     if (mainInput && mainInput.startsWith('/')) {
-      typeArg = 'route';
-      nameArg = mainInput;
+      if (mainInput.endsWith(':md')) {
```

Comment:
just my thought: wouldn't `npm run qwik new /about.md` be more natural? and just thought about `.mdx` as well 👼

Reply:
@zanettin 

I considered that too. I wasn't sure if it would appear as if we'd be creating an `/about.md` Markdown file. I'm okay with `.md` though. I also considered MDX. Should MDX be in this PR or would it be better as a later PR?

Reply:
Changed the `:md` suffix to `.md` and added support for `.mdx`.

Reply:
thank u very much! will test it asap 🙏